Automattic/jetpack
# Components: Move product icons from My Jetpack into components package

Moves product icons into JS components

#### Changes proposed in this Pull Request:

* Moves product icons into components making them available as imports from `@automattic/jetpack-components`.
* Adds README.md to the icon directory

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Checkout this branch, on a connected site
* Visit My Jetpack
* Confirm the icons are render on the product cards.
